specification/status_and_support_level.md

```diff
@@ -0,0 +1,30 @@
+# Status and support levels of open source components
+
+**Status**: [Experimental](../README.md#versioning-and-status-of-the-specification)
+
+## Definitions of status levels
```

Comment:
The status levels definitions are scattered in two places (here and `specification/versioning.md`). It makes me hard to follow.
I think that there are 3 concepts and they should be described in the following order:
  - status levels and development stages of components
  - support levels depending on the component owner and status level
  - versioning (and structuring) depending on component status level
 
I propose to describe everything above in a file named  `component-status.md` or simply `status.md`. I think we can even get rid of `versioning.md` as I do not think we need to call out the specification versioning policy.

Reply:
Based on the conversion it should be spitted into 3 documents:
- `status.md`
- `support.md`
- `versioning.md`

In future we will also need `releasing.md` defining the component releasing requirements.
Changing the PR to draft.